### PR TITLE
Add RunOptimize, to shrink binary size of bitmaps when called

### PR DESCRIPTION
**Pull request by smmathews-bw-boston**

### CODE REVIEW

**Comment by smmathews-bw-boston** [comment]
@acmcmurray uttered this phrase in the other PR: "@bwrobc @bashtanov know C 🧠"

**Review by acmcmurray — APPROVED** [review]
🦁

File: roaringbitmap--0.5--0.6.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* roaringbitmap--0.5--0.6 */
+
+CREATE OR REPLACE FUNCTION rb_runoptimize(roaringbitmap)
```
Comment (bashtanov) [inline roaringbitmap--0.5--0.6.sql:3]: what's the plan? to use this function in updates from time to time?

Comment (smmathews-bw-boston) [reply roaringbitmap--0.5--0.6.sql:3]: yes. the time series team have a script that, currently, every 5 minutes condenses all the raw bitmap data in to a weekly bucket. Optimizing on conflict should allow these bitmaps to be more compact, when possible.

**Pull request merged**